src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs

```diff
@@ -121,7 +121,7 @@ public async Task<IActionResult> GetRecommendations(string sessionId)
                 output.Recommendations.Add(new RecommendationSummary(
                     recommendation.Recipe.Id,
                     recommendation.Name,
-                    recommendation.Description
+                    recommendation.ShortDescription
```

Comment:
Shouldn't we return back both ShortDescription and Description to the toolkit? I through the full Description was going to show up in the detail window or something.

Reply:
We are doing that, but as far as the RecommendationSummary is concerned, it only cares about the short description. The long description is retrieved via the GetRecipe API